frontend/src/components/sightingCard/sightingCard.tsx

```diff
@@ -0,0 +1,41 @@
+import "./sightingCard.scss";
+import { SightingResponse } from "../../clients/internalApiClient";
+import { format, parseISO } from "date-fns";
+
+interface SightingCardProps {
+  sighting: SightingResponse;
+}
+
+export function SightingCard(props: SightingCardProps): JSX.Element {
+  const sighting = props.sighting;
+  const species = props.sighting.species;
+  const formattedDate = format(parseISO(sighting.date), "do MMMM, yyyy");
+
+  return (
+    <div className="sighting-card">
+      <p>Description: {sighting.description}</p>
+      <p>Spotted on: {formattedDate}</p>
+      <p>Latitude: {sighting.latitude}</p>
+      <p>Longitude: {sighting.longitude}</p>
+      {sighting.photoUrl ? (
+        <img src={sighting.photoUrl} alt="sighting of whales" />
+      ) : (
+        <img
+          src="https://i0.wp.com/handluggageonly.co.uk/wp-content/uploads/2017/05/humpback-1209297_1920.jpg?w=1600&ssl=1"
+          alt="sighting of whales"
+        />
```

Comment:
I love this, but I think it might accidentally mislead people to have a "default whale" image - it's not as fun, but I think we should just leave it blank!

Reply:
We have added a different image here, if you still want no image, we can change it.

![image](https://user-images.githubusercontent.com/61541702/177347579-b5d2b25e-5b45-4ab0-b9b4-7e9cefe67192.png)